addons/hr/views/hr_views.xml

```diff
@@ -403,12 +403,20 @@
                                     <strong><field name="name"/></strong>
                                 </div>
                                 <div>
-                                    <span><field name="department_id"/></span>
+                                    <span><field name="department_id"/>&amp;nbsp;</span>
                                 </div>
                                 <div>
                                     <span>Vacancies : <field name="expected_employees"/></span>
-                                    <span class="pull-right label label-default"><field name="state"/>
-                                    </span>
+                                    <t t-if="record.state.raw_value == 'recruit'" >
+                                        <span class="pull-right label label-success">
```

Comment:
Isn't it possible to do a t-att-class and compute the label-something according to the state instead of doing an if-else ?